test/api/controllers/bulkController.test.js

```diff
@@ -286,4 +290,80 @@ describe('Test the bulk controller', () => {
       should(response.deleted).be.eql(2);
     });
   });
+
+  describe('#updateByQuery', () => {
+    let query;
+    let changes;
+    let esResponse;
+
+    beforeEach(() => {
+      query = {
+        match: { foo: 'bar' }
+      };
+      changes = {
+        bar: 'foo'
+      };
+
+      request.input.body = { query, changes };
+
+      esResponse = {
+        updated: 1
+      };
+
+      kuzzle.ask
+        .withArgs('core:storage:public:bulk:updateByQuery')
+        .resolves(esResponse);
+    });
+
+    it('should forward to the store module', async () => {
+      request.input.args.refresh = 'wait_for';
+      const response = await controller.updateByQuery(request);
+
+      should(kuzzle.ask).be.calledWith(
+        'core:storage:public:bulk:updateByQuery',
+        index,
+        collection,
+        query,
+        changes,
+        { refresh: 'wait_for'});
+
+      should(response).be.eql(esResponse);
```

Comment:
It's really nitpicking but usually tests are structured in 3 parts:
 - preparation
 - test
 - verification
 
 It's better to add newline between those parts for readability
 
 ```suggestion
      request.input.args.refresh = 'wait_for';
      
      const response = await controller.updateByQuery(request);

      should(kuzzle.ask).be.calledWith(
        'core:storage:public:bulk:updateByQuery',
        index,
        collection,
        query,
        changes,
        { refresh: 'wait_for'});
      should(response).be.eql(esResponse);
```